### PR TITLE
refactor and update util to use new log formatter setting

### PR DESCRIPTION
**Pull request by brianchennn**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,19 +1,44 @@
...
 	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d
 	github.com/free5gc/aper v1.0.4
 	github.com/free5gc/ngap v1.0.6
-	github.com/free5gc/util v1.0.3
+	github.com/free5gc/util v1.0.4
 	github.com/sirupsen/logrus v1.8.1
 	github.com/urfave/cli v1.22.5
 	github.com/vishvananda/netlink v1.1.0
-	github.com/wmnsk/go-gtp v0.8.0
+	github.com/wmnsk/go-gtp v0.7.28
```
Comment (tim-ywliu) [inline go.mod:14]: should be v0.8.0

File: go.mod [hunk 2]
```diff
@@ -1,19 +1,44 @@
...
+go 1.17
 
 require (
 	git.cs.nctu.edu.tw/calee/sctp v1.1.0
-	github.com/antonfisher/nested-logrus-formatter v1.3.1
 	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d
 	github.com/free5gc/aper v1.0.4
 	github.com/free5gc/ngap v1.0.6
-	github.com/free5gc/util v1.0.3
+	github.com/free5gc/util v1.0.4
```
Comment (tim-ywliu) [inline go.mod:10]: should be v1.0.5-0.20230306071612-a52909216bd2

File: internal/ngap/handler/handler.go [hunk 3]
```diff
@@ -429,126 +421,127 @@ func HandleInitialContextSetupRequest(amf *context.N3IWFAMF, message *ngapType.N
 	var emergencyFallbackIndicator *ngapType.EmergencyFallbackIndicator
 	var iesCriticalityDiagnostics ngapType.CriticalityDiagnosticsIEList
 
-	var n3iwfUe *context.N3IWFUe
+	var n3iwfRanUe *context.N3IWFRanUe
```
Comment (tim-ywliu) [inline internal/ngap/handler/handler.go:424]: replace `n3iwfRanUe` with `ranUe`

**Review by tim-ywliu — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (Bunos07) [reply go.mod]: ✌️

**Pull request merged**